meeting-notes/lcwg-dev-57.md

```diff
@@ -0,0 +1,45 @@
+# Light Client Working Group Devs Meeting 57
+### Meeting Date/Time: Thursday, Aug 17th, 2023 17:00 UTC
+### Meeting Duration: 30 minutes
+### Moderator: @pacu - ZWCD
+### Attendees: @nuttycom (ECC), John (ZF), Matthew Watt (NH), Mandeep Bhalothia (NH), Conrado (ZF), @str4d (ECC), Aditya 
+### Notes: @pacu
+
+### 1. General Notes
+#### Mempool growth and transaction eviction issues
+- Adi is getting reports from users of different wallets about transactions not going through
+- Str4d says the problem might be related to transaction expiry heights not being set properly or
+ZIP-317 not being enforced properly. Recommended expiry height is 40 blocks since Blossom. If a transaction is stuck
+- Reports on graphana show that the mempool started to grow up in size within the previous 15 days. One workaround would be shrinking the mempool size on nodes, setting expiry heights and enforcing ZIP-317 fees.
+- According to NH team, the mempool is being fully populated with small transactions with small amount of actions and the blocks are barely including any of them, therefore causing user txs
+being unmined and expired.
+
+Pacu: What actions that can be taken to avoid workarounds and get the concrete solutions out?
+
+Kris: 
+- Review and get void out for the changes of transaction proposal here: https://github.com/zcash/librustzcash/pull/891
```

Comment:
"void" ?

Reply:
LOL apparently VS Code thought it could autocorrect